tests/sentry/integrations/gitlab/test_issues.py

```diff
@@ -340,3 +357,44 @@ def test_create_issue_no_projects(self):
                 'maxRows': 10,
             }
         ]
+
+    @responses.activate
+    def test_after_link_issue(self):
+        responses.add(
+            responses.POST,
+            u'https://example.gitlab.com/api/v4/projects/2/issues/321/notes',
+            json=[]
+        )
+        data = {'externalIssue': '2#321', 'comment': 'This is not good.'}
+        external_issue = ExternalIssue.objects.create(
+            organization_id=self.organization.id,
+            integration_id=self.integration.id,
+            key='2#321',
+        )
+        self.installation.after_link_issue(external_issue, data=data)
+
+    def test_after_link_issue_required_fields(self):
+        data = {'externalIssue': '2#231', 'comment': 'This is not good.'}
+        external_issue = ExternalIssue.objects.create(
+            organization_id=self.organization.id,
+            integration_id=self.integration.id,
+            key='#',
+        )
+        with pytest.raises(IntegrationError):
```

Comment:
Is this different than using `self.assertRaises` in any way? Not sure where that comes from, tbh, but I've def seen it in this codebase a lot.

Reply:
I'll switch to that.